ackermann_msgs/AckermannDrive drive_command
bool obstacle_detected
